build_for_ultimaker.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/sh
-
+#
+# Copyright (C) 2018 Ultimaker B.V.
+# Copyright (C) 2018 Olliver Schinagl <oliver@schinagl.nl>
+# Copyright (C) 2018 Raymond Siudak <raysiudak@gmail.com>
+#
+# SPDX-License-Identifier: AGPL-3.0+
 set -eu
```

Comment:
missing whitespace

Reply:
Yeah is see that changes from 305 are also in here because i rebased. I undid this and the changes should be in 302, so ignoring for now.